package.json

```diff
@@ -34,8 +35,10 @@
     "new:rule": "node dev/generate.js new-rule",
     "release": "release-it",
     "test": "npm-run-all lint:* test:*",
-    "test:jest": "jest --coverage",
-    "update:readme": "node ./scripts/update-rules.js"
+    "test:jest": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --coverage",
```

Comment:
`--experimental-vm-modules` needed for running ESM tests with jest.

https://jestjs.io/docs/ecmascript-modules